src/tariff-zones-selector/TariffZonesSelectorMap.tsx

```diff
@@ -182,7 +183,7 @@ const TariffZonesSelectorMap = ({
             </MapboxGL.ShapeSource>
             {featureCollection.features.map((f) => (
               <MapboxGL.ShapeSource
-                key={f.id}
+                key={typeof f.id === 'number' ? f.id?.toString() : f.id}
```

Comment:
This could be simplified to `String(id)`. Not important at all, just a tip for later